shared/src/commonMain/kotlin/ui/verify/VerificationComponent.kt

```diff
@@ -23,7 +23,12 @@ class DefaultVerificationComponent(
             return@runBlocking verifyAccount(Data.personId, code.toIntOrNull() ?: return@runBlocking false)
         }
 
-        if (success) {
+        if (success == null){
```

Comment:
dit even veranderen naar success != true (oftewel success is false or success is null)